service/templates/projects.html

```diff
@@ -1,13 +1,15 @@
 {{define "scripts"}}
 <script>
-  window.allTrackedProjects = {{.AllProjects}};
+  window.trackedProjects = {{.AllProjects}};
   {{if .User}}
   window.user = {{.User}};
   window.isAdmin = {{.ProjectData.IsAdmin}};
   {{end}}
   window.canCreate = {{.CanCreate}};
   window.validDefaultLoggers = {{.ValidDefaultLoggers}};
   window.userTz = {{GetTimezone $.User}}
+  alert(trackedProjects.length>0)
```

Comment:
What does this do? Is this for debugging?

Reply:
oh oops! Yes, I'll remove it.